docs/guide/custom-functions.md

```diff
@@ -138,21 +142,23 @@ method:
 HyperFormula.registerFunctionPlugin(MyCustomPlugin, MyCustomPluginTranslations);
 ```
 
-### 6. Use your custom function inside a formula
+### 6. Use your custom function in a formula
 
-Now, you can use your GREET function inside a formula:
+Now, you're ready to use your GREET function in a formula.
+
+Don't forget to register your function plugin **before** you create your HyperFormula instance. Otherwise, HyperFormula won't recognize your plugin.
 
 ```js
-// prepare spreadsheet data
-const data = [['Anthony', '=GREET(A1)']];
+// register your function plugin and its translations
+HyperFormula.registerFunctionPlugin(MyCustomPlugin, MyCustomPluginTranslations);
```

Comment:
This line is already mentioned in step 5. Do you think it's beneficial to duplicate it here?

Reply:
We can do it this way: [58f0eb6](https://github.com/handsontable/hyperformula/pull/1236/commits/58f0eb60fb99ef54efe1f0de0401e633576329a9)

Reply:
Alright, now we have no duplication. Thanks ;)